### PR TITLE
[wg] Update Upgrade testnet with new parameters

### PR DESCRIPTION
**Pull request by Mengerian**
@imaginaryusername Please check for accuracy.

@sickpig @cpacia Please let me know the appropriate commands for BU and BCHD so that they can also be added in.

### CODE REVIEW

**Comment by imaginaryusername** [comment]
To hasten reorg to normal testnet, anyone on the nuked testnet might want to `invalidateblock 000000000b1246b802e56ce28c2beb597b907ca44d983e8b0c71f7f224fd97ab` as well.

**Comment by Mengerian** [comment]
Added extra invalidateblock as per @imaginaryusername comment

File: workgroups/wg-testing/2019-11-15_upgrade_testnet.md [hunk 1]
```diff
@@ -7,34 +7,29 @@ To assist technical preparations for the upgrade, a testnet is available where t
...
-You should also run this to avoid soft-forking back onto normal testnet:
+You should also run these commands to get from the previous upgrade testnet back onto regular testnet,
+and then from regular testnet onto the new upgrade testnet:
 
 ```
-bitcoin-cli -testnet invalidateblock 000000000000067656459385ff54b1178d985a5334f40e209c1e3580c08cc18b
-```
+bitcoin-cli -testnet reconsiderblock 000000000000067656459385ff54b1178d985a5334f40e209c1e3580c08cc18b
+bitcoin-cli -testnet invalidateblock 000000000b1246b802e56ce28c2beb597b907ca44d983e8b0c71f7f224fd97ab
+bitcoin-cli -testnet invalidateblock 000000000002364d1632d3be0744d1e4283b9a478097fb1e115ad98de94cd3a0
```
Comment (sickpig) [inline workgroups/wg-testing/2019-11-15_upgrade_testnet.md]: missing ``` here

File: workgroups/wg-testing/2019-11-15_upgrade_testnet.md [hunk 2]
```diff
@@ -7,34 +7,29 @@ To assist technical preparations for the upgrade, a testnet is available where t
...
+bitcoind -testnet -gravitonactivationtime=1570082400 -addnode=testnet.imaginary.cash
 ```
 
 Alternatively as bitcoin.conf file:
 ```
 testnet=1
 addnode=testnet.imaginary.cash
-gravitonactivationtime=1566227700
+gravitonactivationtime=1570082400
 ```
```
Comment (sickpig) [inline workgroups/wg-testing/2019-11-15_upgrade_testnet.md]: I would add the conf needed also to run alternative clients (BU, BCHD, etc). The BU ones are: 

```
bitcoind -testnet -consensus.forkNov2019Time=1570082400 -addnode=testnet.imaginary.cash
```

and 

```
testnet=1
addnode=testnet.imaginary.cash
consensus.forkNov2019Time=1570082400
```

File: workgroups/wg-testing/2019-11-15_upgrade_testnet.md [hunk 3]
```diff
@@ -7,34 +7,29 @@ To assist technical preparations for the upgrade, a testnet is available where t
...
 
-You should also run this to avoid soft-forking back onto normal testnet:
+You should also run these commands to get from the previous upgrade testnet back onto regular testnet,
+and then from regular testnet onto the new upgrade testnet:
```
Comment (sickpig) [inline workgroups/wg-testing/2019-11-15_upgrade_testnet.md]: I think we should state the options of syncing `testnet3` from scratch as an alternative.

**Pull request merged**